UI/obs-app.cpp

```diff
@@ -1717,7 +1717,9 @@ QString OBSTranslator::translate(const char *context, const char *sourceText,
 				 const char *disambiguation, int n) const
 {
 	const char *out = nullptr;
-	if (!App()->TranslateString(sourceText, &out))
+	std::string str = string(sourceText);
+	str.erase(remove_if(str.begin(), str.end(), isspace), str.end());
+	if (!App()->TranslateString(str.c_str(), &out))
 		return QString(sourceText);
```

Comment:
std::remove_if seems to behave differently on Linux/macOS, do we need to remove all type of whitespaces ?

Source text usually only contains spaces ` `, so only replacing spaces should be fine with whether std::string, dstr or QString.

Example with dstr:
```suggestion
	struct dstr str;
	dstr_init_copy(&str, sourceText);
	dstr_replace(&str, " ", "");
	if (!App()->TranslateString(str.array, &out))
	{
		dstr_free(&str);
		return QString(sourceText);
	}
	dstr_free(&str);
```

Add `dstr_depad(&str);` if you really want to remove other type of whitespace.

Reply:
Just a standard space should be fine in my opinion. I'll test this out. Thanks!